src/powershell/Public/Remove-FinOpsHubScope.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Delete a Cost Management export and optionally data associated with the export.
+
+    .PARAMETER Id
+    Required Resource ID of the scope to remove.
+
+    .PARAMETER HubName
+    Optional. Name of the hub instance to update.
+
+    .PARAMETER RemoveData
+    Optional. Indicates whether to remove data for this scope from storage. Default = false
+
+    .EXAMPLE
+    Remove-FinOpsHubScope -Id "ResourceID of Scope" -HubName "Hub Name" 
+    Remove-FinOpsHubScope -Id "ResourceID of Scope" -HubName "Hub Name" -RemoveData
+    
+    .LINK
+    https://aka.ms/ftk/Initialize-FinOpsHubDeployment
+#>
+
+
+function Remove-FinOpsHubScope {
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$Id,
+        [string]$HubName,
+        [switch]$RemoveData
+    )
+
+    try {
+        # Get all exports for the scope that are pointed to the included storage account and the msexports container
+        $exports = Get-FinOpsCostExport -Scope $Id  
```

Comment:
Don't forget to filter them down to only include the ones targeting this hub! You'll probably need to call `Get-FinOpsHub` to get the storage account ID and then filter on that.